backend/db/satnogs-05.psql

```diff
@@ -0,0 +1,13 @@
+-- Migration add version table for support migration process
+
+CREATE TABLE version (
```

Comment:
There's already table for this called schema. If you don't like the name, rename schema to version.

Reply:
Good point! I change the code to use this table.